### DISCUSSION TITLE
Is there a way to have WindowBuilder reflect a LAF being loaded?

### DISCUSSION THREAD

**Discussion by ForceConstant** [eclipse/windowbuilder#598] [Q&A] [answered]
I am using FlatLAF, and it is understandable that the look and feel is not reflected in the WindowBuilder, but it would be useful if there was some workaround.  Expecially for FONT sizes, colors, etc. Is this possible?

**Comment by ForceConstant** [comment]
Note I tried outputing the LAF as UIManager.put() calls, but WindowBuilder does not reflect this. Any other ways?

**Comment by ptziegler** [comment, marked as answer]
> Note I tried outputing the LAF as UIManager.put() calls, but WindowBuilder does not reflect this. Any other ways?

Just for clarification, you're talking about the LAF that's shown in the Design page, right? And from what it sounds like you're using Swing?

You should be able to add your own LookAndFeels.

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/d0196dbf-4aa8-4222-a510-83687769d4ca)

As a quick test, I added the LookAndFeels from the latest Maven [jar](https://search.maven.org/artifact/com.formdev/flatlaf/3.2.1/jar) and it seems like the e.g. Darcula theme is working.

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/ec8a94ac-3bca-4004-8cac-91914bf98fea)

**Comment by ForceConstant** [comment]
> > Note I tried outputing the LAF as UIManager.put() calls, but WindowBuilder does not reflect this. Any other ways?
> 
> Just for clarification, you're talking about the LAF that's shown in the Design page, right? And from what it sounds like you're using Swing?
> 
> You should be able to add your own LookAndFeels.
> 
> ![image](https://user-images.githubusercontent.com/70652978/274407075-d0196dbf-4aa8-4222-a510-83687769d4ca.png)
> 
> As a quick test, I added the LookAndFeels from the latest Maven [jar](https://search.maven.org/artifact/com.formdev/flatlaf/3.2.1/jar) and it seems like the e.g. Darcula theme is working.
> 
> ![image](https://user-images.githubusercontent.com/70652978/274407766-ec8a94ac-3bca-4004-8cac-91914bf98fea.png)

Ah thanks, I didn't see that ability
